src/commands/publish/preview/mod.rs

```diff
@@ -69,6 +65,36 @@ pub fn preview(
     Ok(())
 }
 
+fn upload_to_preview(
+    project: &Project,
+    user: Option<GlobalUser>,
+    client: &reqwest::Client,
+) -> Result<Preview, failure::Error> {
+    let create_address = match &user {
+        Some(_user) => format!(
+            "https://api.cloudflare.com/client/v4/accounts/{}/workers/scripts/{}/preview",
+            project.account_id, project.name
+        ),
+        None => "https://cloudflareworkers.com/script".to_string(),
+    };
+
+    let script_upload_form = publish::build_script_upload_form(project)?;
+
+    let mut res = client
+        .post(&create_address)
+        .multipart(script_upload_form)
+        .send()?
+        .error_for_status()?;
+
+    let text = &res.text()?;
+    log::info!("Response from preview: {:?}", text);
```

Comment:
Are we sure we want to log the response from preview before or after we parse it with serde? I would think we'd want folks to see the response after parsing succeeds?

Reply:
anywhere in the code you see `log::info!()`, we are planting debug logs. they don't show unless users pass `RUST_LOG=info` as an env var.